src/components/InventoryPage/EmailModal.jsx

```diff
@@ -0,0 +1,241 @@
+        </Span>
+      </Item>
+    </Email>
+  );
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    sendEmail(makeModalContent.makeModalHeader, email, emailTemplate);
+  };
+
+  useEffect(() => {
+    if (status === 'cancel pickup') {
+      setEmailTemp(
+        'Unfortunately, we have CANCELLED your pickup for this day. You can either reschedule your pickup or cancel it altogether. Please provide this information through the Donation Dashboard at this link.',
+      );
+      setMakeModalContent({
+        makeModalHeader: 'Cancel Pickup',
+        makeModalBody:
+          'Unfortunately, we have CANCELLED your pickup for this day. You can either reschedule your pickup or cancel it altogether. Please provide this information through the Donation Dashboard at this link.',
+      });
+    } else if (status === 'request changes') {
+      setEmailTemp(
+        'Dear Patriots and Paws Donor, We have requested changes to your donation form due to reasons listed before. We have listed the items that we don’t accept below. Please remove these items from your donation form so that we can proceed with the donation pickup. Feel free to email us at patriotsandpaws@gmail.com or call us at [pnp number] for more information or assistance.',
+      );
+      setMakeModalContent({
+        makeModalHeader: '[ACTION REQUIRED] Changes Requested',
+        makeModalBody:
+          'Dear Patriots and Paws Donor, We have requested changes to your donation form due to reasons listed before. We have listed the items that we don’t accept below. Please remove these items from your donation form so that we can proceed with the donation pickup. Feel free to email us at patriotsandpaws@gmail.com or call us at [pnp number] for more information or assistance.',
+      });
+    } else if (status === 'approve') {
+      setEmailTemp(
+        'Thank you for filling out the Donation form! We have approved your donation and are working on scheduling a pickup day. Once a pickup day has been picked on our side, you will get an email to approve or reject the scheduled day. Once again, thank you for supporting our veterans!',
+      );
+      setMakeModalContent({
+        makeModalHeader: 'Approve Donation',
+        makeModalBody:
+          'Thank you for filling out the Donation form! We have approved your donation and are working on scheduling a pickup day. Once a pickup day has been picked on our side, you will get an email to approve or reject the scheduled day. Once again, thank you for supporting our veterans!',
+      });
+    } else if (status === 'scheduled') {
+      setMakeModalContent({
+        makeModalHeader: 'Schedule Pickup',
+        makeModalBody:
+          'We have scheduled your donation pickup for February 8th, 2023. Please navigate to the Donation Dashboard using this link in order to accept or reschedule your pickup. If you have any questions or concerns, email patriotsandpaws@gmail.com.',
+      });
+    }
+  }, [status]);
```

Comment:
Let's extract this out into an object mapping the statues to an object with the `modalHeader` and `modalBody`. Then, the useEffect would simply be 
```
setModalContent(mappingObject[status]);
```